packages/terra-divider/src/Divider.jsx

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
 import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
 import 'terra-base/lib/baseStyles';
 import styles from './Divider.module.scss';
 
+const propTypes = {
+  /**
+  * String to be displayed in-line with the divider.
+  */
+  text: PropTypes.string,
+};
+
 const cx = classNames.bind(styles);
 
-const Divider = ({ ...customProps }) => {
+const Divider = (props) => {
+  const { text, ...customProps } = props;
+
   const dividerClassNames = cx([
     'divider',
     customProps.className,
   ]);
 
-  return <hr {...customProps} className={dividerClassNames} aria-hidden="true" />;
+  return <div {...customProps} className={dividerClassNames} aria-hidden="true" content={text} />;
```

Comment:
Let's change `content` attribute to to `data-content`.

Reply:
Updated! [f614149](https://github.com/cerner/terra-core/pull/2296/commits/f6141490bc5feede0c8c51b8f9513ce833957c3d)